apptuit/apptuit_client.py

```diff
@@ -300,13 +309,18 @@ def __init__(self, metric, tags, timestamp, value):
             timestamp: Number of seconds since Unix epoch
             value: value of the metric at this timestamp (int or float)
         """
-        if tags is None or tags == {}:
-            raise ValueError("Ivalid tags: Metric: "+metric+" need minimum one tag.")
         self.metric = metric
         self.tags = tags
         self.timestamp = timestamp
         self.value = value
 
+    def _get_tags_from_environment(self):
+        try:
+            tags_str=environ[APPTUIT_PY_TAGS]
+            tags=json.loads(tags_str)
```

Comment:
use `os.envrion.get` here as well.
Need to handle parsing error from `json.loads`